app/models/ems_refresh/save_inventory_infra.rb

```diff
@@ -303,4 +303,15 @@ def save_storage_files_inventory(storage, hashes)
     deletes = storage.storage_files(true).dup
     save_inventory_multi(:storage_files, storage, hashes, deletes, [:name])
   end
+
+  def save_filesystem_custom_attributes_inventory(parent, hashes, mode = :refresh)
+    return if hashes.nil?
+
+    deletes = case mode
+              when :refresh then nil
+              when :scan    then parent.custom_attributes(true).dup
+              end
+
+    save_inventory_multi(:custom_attributes, parent, hashes, deletes, [:name, :section])
+  end
```

Comment:
ah, it's under :ems_custom_attributes, that's why I didn't notice, why is it not defined as :custom_attributes ? I would like to have the relation clear